test/controllers/workshops_controller_test.rb

```diff
@@ -33,14 +33,19 @@ class WorkshopsControllerTest < ActionDispatch::IntegrationTest
     assert_select 'title', 'Personal Website | Hack Club Workshops'
   end
 
-  test 'passes metadata to analytics on workshop view' do
+  test 'passes correct page tracking metadata' do
```

Comment:
Can you think of a better name for this test? The old behavior was passing metadata to the frontend. The new behavior is actually recording analytics directly in Ruby.